### PR TITLE
chore: add entity data to audit log in order to  show diff

### PR DESCRIPTION
**Pull request by kentakozuka**
Part of https://github.com/bucketeer-io/bucketeer/issues/804

This PR is a fix of https://github.com/bucketeer-io/bucketeer/commit/9259bedcf36bd635942f6603a21e25acf6545ca9 to show diffs of all entities.

### CODE REVIEW

File: proto/event/domain/event.proto [hunk 1]
```diff
@@ -263,8 +265,8 @@ message FeatureCreatedEvent {
 
 message FeatureUpdatedEvent {
   string id = 1;
-  string data = 2;
-  string previous_data = 3;
+  string data = 2 [deprecated = true];
+  string previous_data = 3 [deprecated = true];
```
Comment (kentakozuka) [inline proto/event/domain/event.proto:269]: These will be deprecated.

**Comment by cre8ivejp** [comment]
@kentakozuka, please fix the conflict.

**Comment by kentakozuka** [comment]
@cre8ivejp Resolved conflicts 👍

**Review by cre8ivejp — APPROVED** [review]
Thanks!

**Pull request merged**